.github/workflows/test.yml

```diff
@@ -3,7 +3,7 @@ name: Test
 on:
   push:
     branches: [ master ]
-  pull_request:
```

Comment:
This should be part of the separate PR.

Reply:
That's my feeling as well, but I think it's really up to bakpakin to decide.

I do think that separating makes later history easier to understand.  I know we don't all follow this course of action all of the time though (^^;

Possibly reversion is simpler too?  Not so sure about this point though.